VideoLocker/res/layout/fragment_add_comment.xml

```diff
@@ -19,13 +19,14 @@
             android:paddingBottom="@dimen/widget_margin"
             android:paddingLeft="@dimen/edx_margin"
             android:paddingRight="@dimen/edx_margin"
-            android:paddingTop="@dimen/widget_margin">
+            android:paddingTop="@dimen/widget_double_margin">
```

Comment:
We need the top spacing to be 20dp in any case, if we add the top margin to the included layout and keep the above padding to its previous version, everything will look fine.

But if the response isn't endorsed, the included textView will b `gone` and the top spacing will be 10dp.